cmd/plugins/balloons/policy/balloons-policy.go

```diff
@@ -1526,7 +1526,13 @@ func (p *balloons) pinCpuMem(c cache.Container, cpus cpuset.CPUSet, mems idset.I
 			c.SetCPUShares(int64(cache.MilliCPUToShares(int64(mCpu))))
 		}
 	}
-	if p.bpoptions.PinMemory == nil || *p.bpoptions.PinMemory {
+	// Start from policy-level PinMemory...
+	pinMemory := p.bpoptions.PinMemory == nil || *p.bpoptions.PinMemory
+	// ...and allow override in balloon-type-level PinMemory
+	if (blnDefPinMemory != nil) {
```

Comment:
nit: have been writing C code lately, now haven't we... or then just good ol' habits die really hard...